spec/lib/active_merchant/billing/klarna_gateway_spec.rb

```diff
@@ -157,8 +158,10 @@
       create_profile
 
       expect(SolidusKlarnaPayments::CreateCustomerTokenService).to have_received(:call).with(
-        order: payment.order,
+        email: payment.order.email,
+        address: payment.order.billing_address || payment.order.shipping_address,
```

Comment:
you extracted `payment.order` into a variable in https://github.com/solidusio-contrib/solidus_klarna_payments/pull/34/files#diff-9bc32aaf55cd748994f6d15d831b1237fd3b8b22dd1bd20c958379a052aade3fR265 but not here - would be good to unify

Reply:
Sorry, I saw that comment after the merge. It doesn't look like blocking changes. If you think it needs to be addressed please open an issue or make your own PR.